### PR TITLE
data: joshjgomes

### PR DESCRIPTION
**Pull request by joshjgomes**
## Check List (Check all the applicable boxes) 

- [ X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ X] All new and existing tests passed.
- [ X] This PR does not contain plagiarized content.
- [ X] The title of my pull request is a short description of the requested changes.

## Screenshots

Screen Shot 2023-05-28 at 8 42 21 AM

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7239"></a>

### CODE REVIEW

File: data/joshjgomes.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+	"name": "jOsh",
+	"type": "Personal",
```
Comment (SaraJaoude) [inline data/joshjgomes.json:3]: this is no longer required:
```suggestion
```

**Comment by SaraJaoude** [comment]
You have conflicts in your file, please resolve. This is usually because your fork is not up to date.  

You may wish to have a look at our Editing Guide in Docs: https://linkfree.io/docs/how-to-guides/editing

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by joshjgomes** [comment]
> You have conflicts in your file, please resolve. This is usually because your fork is not up to date.
> 
> You may wish to have a look at our Editing Guide in Docs: https://linkfree.io/docs/how-to-guides/editing

Thank you @SaraJaoude  🫠

**Pull request merged**